### PR TITLE
Rubocop 🤖

### PR DESCRIPTION
**Pull request by pedrofurtado**
@shirts 

I've disabled some cops, to minimize the changes in source code at this moment.

Later, we can enable one by one, and changing the source code to be rubocop-compliant, in differents PRs 🤝

### CODE REVIEW

**Comment by shirts** [comment]
I'd like to use all-or-none when it comes to RuboCop

```yaml
AllCops:
  NewCops: enable
  SuggestExtensions: false
```

Further updates will be forced to follow the rules.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 - Setup of CI with Github Actions
 - Setup of Dockerfile for local tests
+- Setup of Rubocop
```
Comment (shirts) [inline CHANGELOG.md:5]: NIT:

```suggestion
- Setup of RuboCop
```

**Review by shirts — CHANGES_REQUESTED** [review]
I'd like to use all-or-none when it comes to RuboCop

```yaml
AllCops:
  NewCops: enable
  SuggestExtensions: false
```

Further updates will be forced to follow the rules.

**Comment by pedrofurtado** [comment]
> ```yaml
> AllCops:
>   NewCops: enable
>   SuggestExtensions: false
> ```

The two options `NewCops` and `SuggestExtensions` are already defined in RuboCop file 👍

**Comment by shirts** [comment]
@pedrofurtado Apologies for the confusion. I had meant to only have those 3 lines in the RuboCop config file. The lack of defined rules will follow the RuboCop defaults, which I think are more than reasonable for now.

**Comment by pedrofurtado** [comment]
@shirts Don't you prefer to split this changes in 2 PRs? 

The first PR (this one), for just add RuboCop in repo CI without too much file changes. And the second PR (that I can create after this) with the amount of file changes necessary to use all default rules present in Rubocop, which will have much more files for review 👍 

Do you like this approach?

**Comment by pedrofurtado** [comment]
@shirts 🤝 🤓

**Comment by pedrofurtado** [comment]
@shirts Is the PR ok? 🤓

**Comment by shirts** [comment]
@pedrofurtado Like I eluded to above: I'd like to keep the rules scoped to the Ruby community defaults.

**Pull request closed without merging**